src/forms/dropdown/DropdownMenu.tsx

```diff
@@ -45,9 +46,10 @@ export class DropdownMenu extends React.PureComponent<DropdownMenuProps> {
         closeOnBackdrop: true,
         itemsPath: "",
         _level: 0,
+        style: {},
     };
     private renderChildren(): React.ReactNode[] {
-        const {children, itemsPath, onItemClick, _level} = this.props;
+        const {children, itemsPath, onItemClick, _level, style} = this.props;
```

Comment:
let's add spaces here, since the list become bigger
```
const {
  children,
  itemsPath,
  onItemClick,
  _level,
  style,
} = this.props;
```